solutions/usaco-784.mdx

```diff
@@ -64,4 +67,39 @@ int main(){
 
 </CPPSection>
 
+<PySection>
```

Comment:
different idea, so can you put this in a separate `<LanguageSection>` with `# Solution 2` or smth?

Reply:
wait isn't this same idea ? (try firing each lifeguard?)

Reply:
Wait so what should I do for this